hledger/Hledger/Cli/Commands/Close.hs

```diff
@@ -82,14 +82,9 @@ close CliOpts{rawopts_=rawopts, reportspec_=rspec} j = do
     -- should we show the amount(s) on the equity posting(s) ?
     explicit = boolopt "explicit" rawopts
 
-    -- should we preserve cost information ?
-    normalise = case boolopt "show-costs" rawopts of
-                  True  -> normaliseMixedAmount
-                  False -> normaliseMixedAmount . mixedAmountStripPrices
-
     -- the balances to close
     (acctbals,_) = balanceReport rspec_ j
-    totalamt = maSum $ map (\(_,_,_,b) -> normalise b) acctbals
+    totalamt = maSum $ map (\(_,_,_,b) -> b) acctbals
```

Comment:
Maybe no further comment needed here, but for my edification could you say briefly why we don't have to normalise here now ?

Reply:
`normalise` did two things:
1. Strip prices, which has already been done.
2. Normalise the mixed amount, which is done anyway when you call `maSum`.